app/scripts/services/rowMethods.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .factory('rowMethods', function() {
```

Comment:
Since almost all of this is controlling remembering expand/collapse state, I wonder a generic service we could use for that outside the overview.